styles.css

```diff
@@ -89,16 +89,53 @@ article {
     width: 80%;
 }
 
+main {
+    overflow: auto;
+}
+
 a {
     text-decoration: none;
     color: #F77913;
 }
 
+.recipeContainer {
+    width: 50%;
+    padding: 10px;
+    float: left;
+}
+
+.recipe {
+    width: 100%;
+    overflow: auto;
+    background-color: #FFFFFF;
+    border: 1px solid #F77913;
+    border-radius: 5px;
+    padding: 10px;
+    float: left;
```

Comment:
This doesn't need to float within the .recipeContainer. There's nothing else in it to display next to.